### ISSUE TITLE
Interferes with arbirtrary tiled window resizing (Super + Middleclick drag)

### ISSUE THREAD

**Issue by nekohayo** [Fmstrat/wintile#76]
Hi there,
This extension interferes with the ability to finely adjust the split/tiling between two windows using the Super + middleclick GNOME Shell window resizing feature.

Without this extension, it works without problems; but with this extension, it bounces around incontrollably and tries to quarter-split, etc.

**Comment by royrwood** [comment]
I have the same problem.

For example, if I use WinTile to move a window to the left half of the screen, then try to grab the righthand edge of the window to expand it horizontally, the window jumps so that the upper decoration bar of the window is under my mouse cursor.

**Comment by royrwood** [comment]
Here's the sample log output from WinTile when that happens:

```
Feb 20 16:29:46 DELL-9020 gnome-shell[1855]: [WinTile], windowGrabBegin
Feb 20 16:29:46 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:46 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkForMove
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], restoreApp
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], false
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], {"origFrame":{},"row":0,"col":0,"height":2,"width":1}
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], A) mouse - x:1317 y:726
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], A) window - x:70 y:27 w:1245 h:1412
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], A) origFrame - x:447 y:712 w:1545 h:1413
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], Moving window to (544,716), size (1545,1413)
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], B) mouse - x:1317 y:726
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], B) window - x:544 y:716 w:1245 h:1412
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], B) origFrame - x:447 y:712 w:1545 h:1413
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkIfNearGrid
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], mouse - x:1367 y:728
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], monitor - x:0 y:0 w:2560 h:1440 inB:true
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], space - x:70 y:27 w:2490 h:1413
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], window - x:544 y:716 w:1292 h:1412
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], checkIfNearGrid
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], mouse - x:1482 y:732
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], monitor - x:0 y:0 w:2560 h:1440 inB:true
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], space - x:70 y:27 w:2490 h:1413
Feb 20 16:29:47 DELL-9020 gnome-shell[1855]: [WinTile], window - x:544 y:716 w:1406 h:1412
Feb 20 16:29:48 DELL-9020 gnome-shell[1855]: [WinTile], checkIfNearGrid
Feb 20 16:29:48 DELL-9020 gnome-shell[1855]: [WinTile], mouse - x:1482 y:732
Feb 20 16:29:48 DELL-9020 gnome-shell[1855]: [WinTile], monitor - x:0 y:0 w:2560 h:1440 inB:true
Feb 20 16:29:48 DELL-9020 gnome-shell[1855]: [WinTile], space - x:70 y:27 w:2490 h:1413
Feb 20 16:29:48 DELL-9020 gnome-shell[1855]: [WinTile], window - x:544 y:716 w:1411 h:1412
Feb 20 16:29:48 DELL-9020 gnome-shell[1855]: [WinTile], windowGrabEnd
Feb 20 16:29:48 DELL-9020 gnome-shell[1855]: [WinTile], checkIfNearGrid
```

**Comment by rsxchin** [comment]
> For example, if I use WinTile to move a window to the left half of the screen, then try to grab the righthand edge of the window to expand it horizontally, the window jumps so that the upper decoration bar of the window is under my mouse cursor.



That particular issue was fixed in #101
I still need to track down the Super+middle

**Comment by GrylledCheez** [comment]
fixed in #4be5ef6c520540c9a833a22930b318d500050bb8
